crates/transaction-pool/src/pool/best.rs

```diff
@@ -162,6 +164,8 @@ impl<T: TransactionOrdering> crate::traits::BestTransactions for BestTransaction
     }
 }
 
+impl<T: TransactionOrdering> crate::traits::BestTransactionsFilter for BestTransactions<T> {}
```

Comment:
can we try replacing this with a blanket impl

filter for T where T: BestTransaction

Reply:
this makes a lot of sense and seems possible, will add